packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -17,15 +17,33 @@
   margin-top: 16px;
 }
 
+.uswds-usa-form-group--month {
+  margin-right: 16px;
+}
+
+.uswds-usa-form-group--month-input {
```

Comment:
Is this a specific USWDS class from one of their modules? Using the same prefix that they use is confusing to me if it's not one of theirs.

Reply:
@jamigibbs good call - I have updated the names

Reply:
I guess I'm just wondering why did we need to change the name of this class to begin with?

Now I see `usa-form-group--month-input` and `v3-usa-form-group--month-input` both of which are only used in the v3 variation. And `usa-form-group--month-input` isn't even used in the markup anywhere now.

Reply:
Can we just use `usa-form-group--month-input` like it was and just adjust the CSS for it?

Reply:
@jamigibbs PR has been updated to use only original classes - thanks for the nudge